client_certificate.py

```diff
@@ -237,7 +247,7 @@ def exit_if_exist(self, *files):
                 existing_files.append(file)
         if len(existing_files) > 0:
             logging.error('client_certificate: exiting because the following files already exist - {}'.format(' '.join(existing_files)))
-            logging.error('client_certificate: specify CLIENT_CRT_KEY, CAPATH, and CAKEYPATH or empty the directory')
+            logging.error('client_certificate: speficy CLIENT_CRT_KEY, CAPATH, and CAKEYPATH or empty the directory')
```

Comment:
@Sativarsainath-26 typo error

Reply:
Thanks @Ankitha140, updated the code https://github.com/ibm-hyper-protect/secure-build-cli/pull/34/commits/72396177a689856b0746facf11665b998b063f82 :slightly_smiling_face: